### DISCUSSION TITLE
Use case for upload image and display it in view

### DISCUSSION THREAD

**Discussion by DZ8540** [adonisjs/core#3510] [Help] [answered]
Hello there, i want to upload image from form, and i got it, but i don't understand how to display this image from tmp folder.

I saved image with local driver and image has stored in default tmp path.

<img width="446" alt="image" src="https://user-images.githubusercontent.com/50152205/150758603-ae5f5995-10e6-4696-875f-50765dee374c.png">

In DB
<img width="283" alt="image" src="https://user-images.githubusercontent.com/50152205/150758852-682d7e58-2f40-4dec-9fc8-ec1fd167fdb4.png">

In model
<img width="624" alt="image" src="https://user-images.githubusercontent.com/50152205/150759343-cd0d6633-0a70-47f6-887e-8ff0be393fb6.png">

In view
<img width="435" alt="image" src="https://user-images.githubusercontent.com/50152205/150759232-4135fa1e-1468-46e7-b6b5-89b745b5e268.png">

**Comment by arthur-er** [comment]
Computed properties should be getters

**Reply by DZ8540** [reply]
I know. Image from model is just an experiment, but it doesn't solve my problem...

**Reply by thetutlage** [reply]
Are you building an API or using Edge templates?

**Reply by DZ8540** [reply]
Using Edge templates

**Reply by thetutlage** [reply]
There is no need to create any computed property. Just call the function as it is.

```
{{ await modelName.imageUrl() }}
```

**Reply by DZ8540** [reply]
OK, i'm check it later, thank you very much

**Reply by DZ8540** [reply]
> There is no need to create any computed property. Just call the function as it is.
> 
> ```
> {{ await modelName.imageUrl() }}
> ```

Yes it works, thank you very much!

**Comment by DZ8540** [comment, marked as answer]
> There is no need to create any computed property. Just call the function as it is.
> 
> ```
> {{ await modelName.imageUrl() }}
> ```

Try this.
